rtf_vis_tool/src/Components/PCViewer.js

```diff
@@ -61,11 +61,9 @@ function PCViewer(props) {
             arrStringPoints.shift();
         }
 
-        /* Variable pointCloudRaw is an (N x 6) array, with the first 3 entries as (x,y,z) and the last
+        /* Variable arrNumPoints is an (N x 6) array, with the first 3 entries as (x,y,z) and the last
```

Comment:
naming it `arrNumPoints` sounds like it's an array. but it's an int, i believe. How about `num_points`? or `n_points`?

Reply:
So actually `arrNumPoints` is supposed to be an (N x 6) array, but the name I have for it is actually misleading. I'll rename it to `arr_points`.